contracts/ics-escrow/.gitignore

```diff
@@ -0,0 +1,15 @@
+# Build results
```

Comment:
How many of these do we need? :joy:

Reply:
Consolidated all of them into the root. :)